app/src/main/scala/org/alephium/explorer/persistence/queries/TransactionQueries.scala

```diff
@@ -56,10 +56,10 @@ trait TransactionQueries extends StrictLogging {
     if (transactions.isEmpty) {
       DBIOAction.successful(0)
```

Comment:
Similar to above. With `QuerySplitter.splitUpdates` this check is no longer needed?